decode.go

```
@@ -134,7 +154,10 @@ func UnmarshalTable(t *ast.Table, v interface{}) (err error) {
 		case *ast.Table:
 			fv, fieldName, found := findField(rv, key)
 			if !found {
-				return fmt.Errorf("line %d: field corresponding to `%s' is not defined in `%T'", av.Line, key, v)
+				if l := getLogger(); l != nil {
```

If this is `nil` should we return the error?